jumanji/environments/routing/connector/env.py

```diff
@@ -68,24 +65,22 @@ class Connector(Environment[State]):
               the bottom right corner and is aiming to get to the middle bottom cell. Agent 2
               started in the top left and moved down once towards its target in the bottom left.
 
-              This would just be agent 0's view, the numbers would be flipped for agent 1's view.
-              So the full observation would be of shape (2, 3, 3).
-
     - action: jax array (int32) of shape (num_agents,):
         - can take the values [0,1,2,3,4] which correspond to [No Op, Up, Right, Down, Left].
         - each value in the array corresponds to an agent's action.
 
     - reward: jax array (float) of shape ():
-        - dense: each agent is given 1.0 if it connects on that step, otherwise 0.0. Additionally,
-            each agent that has not connected receives a penalty reward of -0.03.
+        - dense: reward is increased by 1 for each successful connection on that step. Additionally,
+            each pair of points that have not connected receives a penalty reward of -0.03.
 
     - episode termination: if an agent can't move, or the time limit is reached, or the agent
         connects to its target, it is considered done. Once all agents are done, the episode
-        terminates. The timestep discounts are of shape (num_agents,).
+        terminates. The timestep discounts are of shape (1,) and only set to `0` when all agents
+        are done.
```

Comment:
Is it still relevant to mention the discount here ? I think it is not mentioned in other single-agent envs. 
Maybe bullet points with each ending condition would be more clear for the episode termination ? like in other envs (snake or jobshop for example)